amundsen_application/static/js/interfaces/Analytics.ts

```diff
@@ -0,0 +1,4 @@
+export interface AnalyticsEvent {
+  name: string;
+  properties: { [prop: string]: any };
```

Comment:
Nit: I think that 'unknown' would work as well and would avoid adding an 'any'

Reply:
Ah, good point, changed.